src/Indicator.vala

```diff
@@ -114,6 +128,12 @@ public class Slingshot.Indicator : Wingpanel.Indicator {
         }
 
         indicator_grid.tooltip_markup = Granite.markup_accel_tooltip (accels, _("Open and search apps"));
+
+        if (gala_behavior_settings != null) {
+            if ("wingpanel" in gala_behavior_settings.get_string ("overlay-action")) {
+                indicator_grid.tooltip_markup += ", ⌘";
+            }
+        }
```

Comment:
Thanks for fixing the granite util! Now we can add `<Super>` to accels instead of manually adding to the string

Reply:
Yep, feels much cleaner!